test/util/WiremockTestServer.scala

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package connectors
+package util
```

Comment:
if test/util is a source root, you should have a package inside it e.g. `connectors` and leave the above.

Otherwise the `package` refers to a directory, and not an actual package